### ISSUE TITLE
Cooldown configuration for doorbell events

### ISSUE THREAD

**Issue by vincer** [hjdhjd/homebridge-unifi-protect#984]
**Is your feature request related to a problem? Please describe:**

When someone repeatedly rings the doorbell it causes rings to “stack” on HomePod and when Siri recognizes a person she will announce “it looks like <first name, last name> is at the door”. This announcement will repeat fully over and over as many times as the button was pressed which can take a couple minutes and drive everyone crazy.

My 3 year old, however, finds great amusement in this.

**Describe the solution you'd like:**

A configuration setting allowing for X seconds of cooldown where a doorbell event isn’t sent.
Basically the same as the one that already exists for motion events.

**Describe alternatives you've considered:**

Yelling “Siri, stop!” across the room (which didn’t work but that may have been because of the other chaos). Unplugging HomePod definitely works.

**Additional context:**

**Comment by hjdhjd** [comment]
...a version of this already exists. 😄 

In the HBUP webUI, go to the settings tab, under advanced settings. It's called "ring duration" and dictates the length of each ring event, which defaults to three seconds. You can take it up to ten seconds, if you prefer - meaning HBUP will only process one ring event every ten seconds.

Enjoy!

**Comment by vincer** [comment]
Thanks for the quick response! I’d forgotten there are more settings there. 🤦 

Unfortunately, even after upping it to 10s and restarting homebridge, I’m getting multiple announcements on the HomePod if I quickly ring the doorbell. Interestingly, though, I only got one notification on my phone… is there some other type of event that gets sent on a doorbell press?

**Comment by hjdhjd** [comment]
I understand the request better - stay tuned. I think it’s a great feature addition…coming soon. Thanks!

**Comment by hjdhjd** [comment]
Implemented in v6.15. Enjoy.

**Comment by vincer** [comment]
Works like a charm! Thanks!
